bin/check-services

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
I wrote all the scripts in Python because it's just so much easier to write. Maybe they'd be a bit faster in bash? Not sure. They'd certainly be much harder to write